js_modules/dagster-ui/packages/ui-core/src/partitions/DimensionRangeWizard.tsx

```diff
@@ -113,6 +204,39 @@ export const DimensionRangeWizard = ({
   );
 };
 
+const getMissingPartitions = (health: PartitionStatusHealthSource, partitionKeys: string[]) => {
+  if ('ranges' in health) {
+    const missingRangeTerms = health.ranges
+      .filter((range) => range.value.includes(AssetPartitionStatus.MISSING))
+      .map((range) => ({type: 'range' as const, start: range.start.key, end: range.end.key}));
+    const missingRangeSelections = convertToPartitionSelection(missingRangeTerms, partitionKeys);
+    if (missingRangeSelections instanceof Error) {
+      return [];
+    }
+    return missingRangeSelections.selectedKeys;
+  }
+  return partitionKeys.filter(
+    (key, idx) => health.runStatusForPartitionKey(key, idx) === RunStatus.CANCELED,
```

Comment:
This looks good but this final check here feels extra - I'm not sure that we apply this check in other places we filter to missing partitions. Just looking at the range AssetPartitionStatus might be sufficient?